cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/aggregation/NGSIGenericAggregator.java

```diff
@@ -393,6 +394,24 @@ public void setEntityType(String entityType) {
     } //setEntityType
 
     /**
+     * Gets the entityId
```

Comment:
remove this lines

Reply:
fixed in b179bd50cd3a0408217c4d5d0878910cc8530ce1